packages/core/src/events/types.ts

```diff
@@ -1,8 +1,7 @@
-import { LogEntry, LogEntryQueue } from '../queues';
-import { With } from '../types';
+import { LogEntry } from '../logger';
 
 export interface Events {
-  'queue.push': (entry: LogEntry, queue: LogEntryQueue) => void;
-  'queue.flush': (queue: With<LogEntryQueue, 'id'>) => void;
+  'queue.push': (entry: LogEntry) => void;
+  'queue.flush': (processId: string) => void;
   'queue.write': (url: string) => void;
 }
```

Comment:
trochu přemýšlím, jestli by nestálo za to tyhle eventy přejmenovat - žádná "queue" už tam interně "není" (resp. není na globální úrovni), co myslíš?

Reply:
spis si rikam, ze bych shodil cely eventDispatcher, protoze se nikde nepouziva

Reply:
well.. ok, asi klidne..